server/apps/rules/routing_rules.py

```diff
@@ -140,6 +143,10 @@ class RoutingRuleSchemeResource(Resource):
                             },
                             'hour_of_day_to': {
                                 'type': 'string'
+                            },
+                            'time_zone': {
+                                'type': 'string',
+                                'nullable': True
```

Comment:
if we want to be able to use server local time then `nullable: True` makes sense, but would probably use `default: 'UTC'` anyhow and let client set explicitly if user wants to use server local time. but not sure if it's something we want to support..